docs/xxopenj9commandlineenv.md

```diff
@@ -0,0 +1,40 @@
+<!--
+* Copyright (c) 2017, 2022 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:\[+|-\]OpenJ9CommandLineEnv
+
+This option controls whether the VM captures the command line in the environment variable `OPENJ9_JAVA_COMMAND_LINE`.
+
+## Syntax
+
+        -XX:[+|-]OpenJ9CommandLineEnv
+
+| Setting                     | Effect  | Default on z/OS                                                                      | Default on other platforms                                                           |
+|-----------------------------|---------|:------------------------------------------------------------------------------------:|:------------------------------------------------------------------------------------:|
+| `-XX:+OpenJ9CommandLineEnv` | Enable  |                                                                                      | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
+| `-XX:-OpenJ9CommandLineEnv` | Disable | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |                                                                                      |
+
+It is enabled by default on most platforms. On z/OS, however, it is disabled by default because it is believed to be the cause of at least one crashing scenario. For more information, see [Environment variables](env_var.md).
```

Comment:
I don't think we need "It is enabled by default on most platforms." because that info is covered by the table (and more specifically).
Need to use "this option" instead of the ambiguous "it", z/OS needs a trademark (`&reg;`), and not sure about the problem text. Suggest "This option is currently disabled by default on z/OS&reg; because it might cause the VM to crash." (I used the word "currently" to reassure users that the problem will be fixed, which I assume is the case.)

Rather than "For more information, see <env var topic>", which doesn't really contain much more info (and no more with these review updates), I suggest a "See also" section similar to  https://www.eclipse.org/openj9/docs/xdisablejavadump/#see-also

Presumably there's no way for a user to get the command-line info on z/OS but if IBM_JAVA_COMMAND_LINE wasn't used by OpenJ9 before anyway, I guess there's no change so nothing to document. I assume that's also the reason we're not announcing that IBM_JAVA_COMMAND_LINE is deprecated, or mentioning that it's been removed.

Reply:
IBM_JAVA_COMMAND_LINE is not deprecated or removed, but it's only supported by IBM Java 8, and was never a feature of OpenJ9 builds.